crates/bin/pindexer/src/lqt/schema.sql

```diff
@@ -1,32 +1,179 @@
-CREATE TABLE IF NOT EXISTS lqt_votes (
-    id SERIAL PRIMARY KEY,
+CREATE SCHEMA lqt;
+
+CREATE TABLE lqt._params (
+  epoch INTEGER PRIMARY KEY,  
+  delegator_share NUMERIC(3, 2) NOT NULL,
+  gauge_threshold NUMERIC(3, 2) NOT NULL
+);
+
+CREATE TABLE lqt._finished_epochs (
+    epoch INTEGER PRIMARY KEY
+);
+
+CREATE TABLE lqt._available_rewards (
+  epoch INTEGER PRIMARY KEY,  
+  amount NUMERIC NOT NULL
+);
+
+CREATE TABLE lqt._delegator_rewards (
```

Comment:
One latent question, why are we prefixing the tables with `_`, guess is that they are internals? but if they are used by the view, then they should probably not be right?

Reply:
They are internals from the perspective of consumers of the AppView. Think of the AppView like a class / module, with the private members being the `_` tables, and the public members being the views. The public members can be constructed by reading the internal members, but outside consumers should only consume the interface via the public views.